product_multi_image/models/product_product.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3).
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2014 Serv. Tecnol. Avanzados Pedro M. Baeza
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# © 2016 Pedro M. Baeza, Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _name = "product.product"
+    _inherit = [_name, "base_multi_image.owner"]
+
+    image = fields.Binary(
+        related='image_main',
+        store=False,
+        multi=False
+    )
+    image_medium = fields.Binary(
+        related='image_main_medium',
+        store=False,
+        multi=False
+    )
+    image_small = fields.Binary(
+        related='image_main_small',
+        store=False,
+        multi=False
```

Comment:
I'm afraid these `multi` probably don't have the desired effect.
Can you confirm please? Review https://github.com/odoo/odoo/issues/10799#issuecomment-241704243